plugins/aws/autoscaling/webTierAsgCloudwatchLogs.js

```diff
@@ -0,0 +1,132 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Web-Tier Auto Scaling Group CloudWatch Logs Enabled',
+    category: 'AutoScaling',
+    description: 'Ensures that Web-Tier Auto Scaling Groups are using CloudWatch Logs agent.',
+    more_info: 'EC2 instance available within web-tier Auto Scaling Group (ASG) should use an AWS CloudWatch Logs agent to monitor, store and access log files.',
+    link: 'https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Install-CloudWatch-Agent.html',
+    recommended_action: 'Update web-tier Auto Scaling Group to use CloudWatch Logs agent',
+    apis: ['AutoScaling:describeAutoScalingGroups', 'AutoScaling:describeLaunchConfigurations', 'STS:getCallerIdentity'],
+    settings: {
+        web_tier_tag_key: {
+            name: 'Auto Scaling Web-Tier Tag Key',
+            description: 'Web-Tier tag key used by Auto Scaling groups to indicate Web-Tier groups',
+            regex: '^.*$',
+            default: 'web_tier'
+        },
+        cw_log_agent_install_command: {
+            name: 'Cloudwatch Agent Install Command',
+            description: 'Commands to install Cloudwatch Agent',
+            regex: '^.*$',
+            default: '#!/bin/bash curl https://s3.amazonaws.com//aws-cloudwatch/downloads/latest/awslogs-agent-setup.py -O ' +
+            'chmod +x ./awslogs-agent-setup.py ' +
+            './awslogs-agent-setup.py -n -r <AWS_REGION> -c <S3_CLOUDWATCH_AGENT_CONFIG_FILE_LOCATION>'
+        },
+        s3_cw_agent_config_file: {
+            name: 'S3 Cloudwatch Agent Config File Location',
+            description: 'S3 path of cloudwatch agent configuration file',
+            regex: '^.*$',
+            default: ''
+        }
+    },
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var regions = helpers.regions(settings);
+
+        var config = {
+            web_tier_tag_key: settings.web_tier_tag_key || this.settings.web_tier_tag_key.default,
+            cw_log_agent_install_command: settings.cw_log_agent_install_command || this.settings.cw_log_agent_install_command.default,
+            s3_cw_agent_config_file: settings.s3_cw_agent_config_file || this.settings.s3_cw_agent_config_file.default
+        };
```

Comment:
@giorod3 - somewhere around this line we can also do a check for
```
if (!web_tier_tag.length) return callback()
```
to exit the plugin without doing any checks